src/Drivers/nlpMDS_ex4.c

```diff
@@ -307,17 +307,17 @@ int main(int argc, char **argv) {
 
   // println("ns: $ns, nd: $nd")
 
-  double* Q = malloc(nd*nd*sizeof(double));
+  double* Q = (double*)malloc(nd*nd*sizeof(double));
   for(i=0; i<nd*nd; i=i+1) Q[i] = 1e-8;
   for(i=0; i<nd; i=i+1) Q[i + i*nd] += 2.0;
   for(i=1; i<nd-1; i=i+1) {
     Q[i+1 + i*nd] += 1.0;
     Q[i + (i+1)*nd] += 1.0;
   }
 
-  double* Md = malloc(ns*nd*sizeof(double));
+  double* Md = (double*)malloc(ns*nd*sizeof(double));
```

Comment:
Should we use space between cast expression and the `malloc` function?
```suggestion
  double* Md = (double*) malloc(ns*nd*sizeof(double));
```
If so, perhaps we should update developer guidelines accordingly?

Reply:
I don't really care either way, @cnpetra do you have an opinion?

Reply:
looks better with the space, but it's up to you